src/AddTodoForm.jsx

```diff
@@ -1,10 +1,18 @@
 import * as React from 'react';
 
-function AddTodoForm() {
+function AddTodoForm(props) {
+  const handleAddTodo = (event) => {
+    event.preventDefault();
+    const todoTitle = event.target.elements.title.value;
+    console.log(todoTitle);
+    props.onAddTodo(todoTitle);
+    event.target.reset();
+  };
+
   return (
-    <form>
+    <form onSubmit={handleAddTodo}>
       <label htmlFor="todoTitle">Title</label>
```

Comment:
htmlFor is set to "todoTitle" when it should be set to "title" to match the input `name` attribute. 

See this mdn article for an explanation on associating label and input elements. If you have any questions let me know and Id be happy to set up a session for us to chat on this.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label